components/Navbar.vue

```diff
@@ -72,9 +75,49 @@ import PrefixMixin from '~/utils/mixins/prefixMixin'
   },
 })
 export default class NavbarMenu extends mixins(PrefixMixin) {
+  mounted() {
+    onApiConnect(async (api) => {
+      this.loadBalance(api, this.account)
+    })
+  }
+
   get isRmrk(): boolean {
     return this.urlPrefix === 'rmrk' || this.urlPrefix === 'westend'
   }
+
+  get account(): string {
+    return this.$store.getters.getAuthAddress
+  }
+
+  async loadBalance(
+    api: ApiPromise = Connector.getInstance().api,
+    account: string
+  ): Promise<void> {
+    if (!this.account || !api) {
+      return
+    }
+
+    try {
+      const result = await api.query.system.account(account)
+      const balance = result.data.free.toString()
+      this.$store.dispatch('setBalance', balance)
+
+      // Here we subscribe to any balance changes and update the on-screen value
```

Comment:
@yangwao got to know that we can listen for balance changes from here

https://polkadot.js.org/docs/api/examples/promise/listen-to-balance-change